assets/src/edit-story/elements/text/layer.js

```diff
@@ -36,7 +36,8 @@ const TextLayer = styled.span`
 function TextLayerContent({ element: { content } }) {
   // Remove all tags
   const rawContent = content.replace(/<[^>]*>/g, '');
-  return <TextLayer>{rawContent}</TextLayer>;
+  const ariaLabel = `Text element ("${rawContent}")`;
+  return <TextLayer aria-label={ariaLabel}>{rawContent}</TextLayer>;
```

Comment:
Is the `aria-label` needed for this though? If it's the same as the content anyway?

Reply:
It's the same content but preceeded by the text `"text element"`. It could also be an invisible element in front with the `aria-label="Text element"`.

Reply:
Hm, actually the icon should be the "text element" and the text just itself. Also for the other elements. I'll change that!

Reply:
Much better now in 2f08257.